pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -43,5 +46,18 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool,
 	}
 
 	token := parts[1]
-	return a.auth.AuthenticateToken(token)
```

Comment:
no error from the auth chain meant that it fell through to anonymous auth and got a 403